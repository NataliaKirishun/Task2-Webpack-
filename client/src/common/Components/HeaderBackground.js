import styled from 'styled-components';
import Theme from '../Theme/Theme';
import bckgImage from '../../assets/images/films.jpeg';


const HeaderBackground = styled.div`
    background-color: ${Theme.colors.dark_grey};
    background-image: url(${bckgImage});
    background-repeat: no-repeat;
    background-blend-mode: overlay;
`;

export default HeaderBackground;
