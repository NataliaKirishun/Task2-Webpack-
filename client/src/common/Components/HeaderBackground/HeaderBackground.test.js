import React from 'react';
import { shallow } from 'enzyme';
import HeaderBackground from './HeaderBackground';

describe('HeaderBackground Snapshot', () => {
    it('HeaderBackground Component should render correctly', () => {
        const component = shallow(<HeaderBackground />);
        expect(component).toMatchSnapshot();
    });
});