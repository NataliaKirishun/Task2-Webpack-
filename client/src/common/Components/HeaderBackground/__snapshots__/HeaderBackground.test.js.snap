// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderBackground Snapshot HeaderBackground Component should render correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
    background-color: ",
          "#202020",
          ";
    background-image: url(",
          "process: function process(src, filename, config, options) {
    return 'module.exports = ' + JSON.stringify(path.basename(filename)) + ';';
  };",
          ");
    background-repeat: no-repeat;
    background-blend-mode: overlay;
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
